content/developers/api-reference/events-api/index.md

```diff
@@ -62,457 +55,71 @@ ASSET_UUID=<ASSET_UUID>
-
-```bash
-curl -g -X GET \
-     -H "@$HOME/.datatrails/bearer-token.txt" \
-     "https://app.datatrails.ai/archivist/v2/assets/-/events?event_attributes.arc_display_type=Software%20Package%20Release"
-```
-
-#### Fetch Event by Asset Attribute
-
-To fetch all Events of a specific Asset attribute, `GET` the Events resource and filter on `asset_attributes` at the Asset level:
-
-```bash
-curl -g -X GET \
-     -H "@$HOME/.datatrails/bearer-token.txt" \
-     "https://app.datatrails.ai/archivist/v2/assets/-/events?asset_attributes.document_status=Published"
-```
-
-#### Fetch Events by Filtering for Presence of a Field
-
-To fetch all Events with a field set to any value, `GET` the Events resource and filter on most available fields. For example:
-
-```bash
-curl -g -X GET \
-     -H "@$HOME/.datatrails/bearer-token.txt" \
-     "https://app.datatrails.ai/archivist/v2/assets/-/events?event_attributes.arc_display_type=*"
-```
-
-Returns all Events which have `arc_display_type` that is not empty.
-
-#### Fetch Events Which are Missing a Field
-
-To fetch all Events with a field which is not set to any value, `GET` the Events resource and filter on most available fields. For example:
-
-```bash
-curl -g -X GET \
-     -H "@$HOME/.datatrails/bearer-token.txt" \
-     "https://app.datatrails.ai/archivist/v2/assets/-/events?event_attributes.arc_display_type!=*"
-```
+{{< note >}}
+**Note:** The current preview limits fetching Events to the Event identity.
+Querying across event attributes and trails are coming in a future preview.
+{{< /note >}}
 
-Returns all Events which do not have `arc_display_type` or in which `arc_display_type` is empty.
+#### Fetch Events by Identity
 
-#### Fetch Events by Minimum Confirmation Status
+- Replace the `<event-id>` below, using the event-id from the created event above: `"identity": "events/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"`:
 
-To fetch all Events with a specified confirmation status or higher, `GET` the Events resource and filter on `minimum_trust`.
-For example:
+  ```bash
+  EVENT_ID=<event-id>
+  ```
 
-```bash
-curl -g -X GET \
-     -H "@$HOME/.datatrails/bearer-token.txt" \
-     "https://app.datatrails.ai/archivist/v2/assets/-/events?minimum_trust=COMMITTED"
-```
+- Query the /events API to retrieve the recorded event:
 
-Returns all Events which have a `confirmation_status` level of COMMITTED, CONFIRMED or UNEQUIVOCAL.
+  ```bash
+  curl -X GET \
+      -H "@$HOME/.datatrails/bearer-token.txt" \
+      "https://app.datatrails.ai/archivist/v1/events/$EVENT_ID" | jq
+  ```
 
-```bash
-curl -g -X GET \
-     -H "@$HOME/.datatrails/bearer-token.txt" \
-     "https://app.datatrails.ai/archivist/v2/assets/-/events?minimum_trust=CONFIRMED"
-```
+## Events OpenAPI Docs
 
-Returns all Events which have a `confirmation_status` level of CONFIRMED or UNEQUIVOCAL.
+{{< openapi url="https://raw.githubusercontent.com/datatrails/datatrails-openapi/master/doc/events.swagger.json" >}}
```

Comment:
This line is failing within github actions, which makes sense as the file isn't posted yet.
However, locally, hugo resolves this line, so I'm a bit confused how this gets resolved locally, vs. remotely

Reply:
this is fixed now